GenderPayGap.WebUI/Content/site.charts.css

```diff
@@ -101,6 +103,8 @@ govuk-chart-vert__head {
     width: 4em;
     height: 100%;
     bottom: 0;
+    outline: 2px solid;
+    outline-color: transparent;

```

Comment:
Any particular reason why in some we do `2px solid transparent` but here we do `2px solid; outline-color: transparent`?